python/lsst/ts/cbp/component.py

```diff
@@ -1,140 +1,119 @@
-    The following api exposes commands that move the motors of the CBP, sets the focus and selects the mask.
-
-    Parameters
-    ----------
-    address : str
-       The ip address of the CBP galil controller.
-    port : int
-        The port to connect to the CBP galil controller.
+    The component implements a python wrapper over :term:`DMC` code written by
+    DFM Manufacturing.
+    The following api exposes commands that move the motors of the CBP, sets
+    the focus and selects the mask.
 
     Attributes
     ----------
-    log: logging.Logger
+    log : `logging.Logger`
         The logger for the component
 
-    socket: socket.Socket
+    socket : `socket.Socket`
         The socket that handles the TCP/IP connection for the CBP
 
-    altitude: float
-        The value of the CBP altitude encoder in degress.
+    altitude : `float`
+        The value of the CBP altitude encoder in degrees.
 
-    azimuth: float
+    azimuth : `float`
         The value of the CBP azimuth encoder in degrees.
 
-    mask: str
-        The current mask as named in the mask_dictionary
+    mask : `str`
+        The current mask name
 
-    mask_rotation: float
+    mask_rotation : `float`
         The current value of the mask rotation encoder in degrees.
 
-    masks: SimpleNamespace
-        A simplenamespace that contains the mask names and rotation values along with the id.
+    masks : `SimpleNamespace`
+        A simplenamespace that contains the mask names and rotation values and
+        id.
 
-    focus: float
+    focus : `float`
         The current value of the focus encoder in microns.
 
-    panic_status: float
+    panic_status : `float`
         The current value of the panic variable in the CBP dmc code.
-        A non-zero value represents a panic state and causes the motors to cease functioning until panic is
-        dealt with or goes away.
-        This status is related to the other statuses.
+        A non-zero value represents a panic state and causes the motors to
+        cease functioning until panic is dealt with or goes away.
+        This status is set by the values of the other statuses.
 
-    auto_park: float
+    auto_park : `float`
         The current value of the auto_park variable.
-        If this value is one, that means that CBP suffered a power loss that lasted more than 12 seconds and
-        was on battery back up.
-        The CBP will then park itself automatically, moving azimuth to 0 and altitude to -70 and lock focus
-        and mask.
-        To unpark CBP, the park variable should be set to zero.
+        If this value is one, that means that CBP suffered a power loss that
+        lasted more than 12 seconds and was on battery back up.
+        The CBP will then park itself automatically, moving azimuth to 0 and
+        altitude to -70 and lock focus and mask.
+        To un-park CBP, the park variable should be set to zero.
```

Comment:
Is this still true? I thought you were going to switch to calling a method to unpark.

Reply:
Ping

Reply:
I'd like to switch the method in the xml to use park and un-park then change it over in the component class.